Sources/Sentry/Public/SentryDebugImageProvider.h

```diff
@@ -1,6 +1,8 @@
+#import "SentryDebugMeta.h"
+#import "SentryThread.h"
 #import <Foundation/Foundation.h>
 
-@class SentryDebugMeta, SentryThread, SentryFrame;
```

Comment:
Forward declarations also didn't work in this and similar cases and were replaced with imports.